utils/logger.ts

```diff
@@ -0,0 +1,7 @@
+import { SHOW_DEBUG_LOGS } from '../app';
+
+export function log(content: string) {
```

Comment:
don't you want a log level here and have some basic logs as info and the more verbose as debug? I would like to know e.g. how many issues the service found at the end of a run